docs/_sections/forms.html

```diff
@@ -87,6 +87,34 @@ <h3 id="requiredFields">Required fields</h3>
 &lt;/div&gt;
     </code></pre>
 
+    <h3 id="fieldErrors">Error state on fields</h3>
+
+    <p>Any field where user input could be deemed or returned erroneous should utilize the class <code>nypl-field-error</code>. It is up to the form designer to determine what fields should have an error state. It is up to the developer to implement the necessary front and back end validations as well as the expected input formatting that would necessitate a state change for a given field. Additionally, explanatory text must be supplied by the form designer decribing the error and how to correct it. </p>
+
+    <div class="nypl-toolkit-example">
+      <span class="nypl-example-text">Example</span>
+      <div class="nypl-text-field nypl-field-error">
+        <label for="keyword-text">Title
+          <span class="nypl-required-field">Required</span>
+        </label>
+        <input id="keyword-text" type="text" placeholder="User's erroneous input" />
```

Comment:
provide an example: like the field being Email, the email written being “sally ride”, and the error message being “The email is invalid. Please provide a valid email.”